addons/mail/static/src/js/owl/store/messaging_actions.js

```diff
@@ -807,12 +809,36 @@ const actions = {
         if (authorPartnerId === currentPartner.id) {
             return;
         }
+
         const threadLocalId = `mail.channel_${channelId}`;
         const thread = state.threads[threadLocalId];
+        if (!env.call('bus_service', 'isOdooFocused')){
+            dispatch('_handleNotificationOutOfFocusMessage', message);
```

Comment:
don't use message, use message local id and get message body and author from store